server/api/controllers/rpaFrameworkCommandsController.js

```diff
@@ -33,7 +76,24 @@ exports.getAvailableTasksForApplications = async (req, res) => {
   }
 };
 
-// GET /rpa-framework/commands/getAllParameters
+/**
+ * @swagger
+ * /functionalities:
+ *     get:
+ *       tags:
+ *         - RPA-Functionalities
+ *       summary: Get all available Task and Application combinations with the input parameters and possible output values
+ *       operationId: getFunctionalities
+ *       responses:
+ *         200:
+ *           description: OK
+ *           content:
+ *             application/json:
+ *               schema:
+ *                 type: array
+ *                 items:
+ *                   $ref: '#/components/schemas/Functionalities'
+ */
 exports.getAllParameters = async (req, res) => {
```

Comment:
I would probably rename this getAllParameters function as I think the name is kind of misleading. Perhaps something like getAllRpaFunctionalities ?